src/lms-ds-loader/edfi_lms_ds_loader/loader_facade.py

```diff
@@ -132,23 +114,16 @@ def _load_assignments(csv_path: str, db_adapter: MssqlLmsOperations) -> None:
         sections_df["SourceSystemIdentifier"]
     )  # we only need to access the last file
 
-    def __get_all_file_paths_callback():
-        formatted_path = os.path.abspath(csv_path)
-        paths = []
-        for section in sections:
-            paths = paths + file_repository.get_assignments_file_paths(
-                formatted_path, section
-            )
+    formatted_path = abspath(csv_path)
+    file_paths: List[str] = []
+    for section in sections:
+        file_paths = file_paths + file_repository.get_assignments_file_paths(
+            formatted_path, section
+        )
```

Comment:
It is bugging me that I can't recall why this was done with a callback originally. There was a reason for it at the time.